Sources/Orbit/Support/Environment Keys/HapticsEnabledKey.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftUI
+
+struct HapticsEnabledKey: EnvironmentKey {
+    static let defaultValue = true
+}
+
+public extension EnvironmentValues {
+
+    /// Whether haptic feedback on controls is enabled.
+    var isHapticsEnabled: Bool {
+        get { self[HapticsEnabledKey.self] }
+        set { self[HapticsEnabledKey.self] = newValue }
+    }
+}
+
+public extension View {
+
+    /// Disables haptic feedback on controls.
+    func disabledHaptics(_ disabled: Bool = true) -> some View {
```

Comment:
Suggestion - the Apple API seems to have switched naming for such cases. We could go with `hapticsDisabled()`

<img width="631" alt="image" src="https://github.com/kiwicom/orbit-swiftui/assets/35844477/19220416-66b4-49e4-b24a-54100ab469ed">

Reply:
Updated.